static/src/js/index.jsx

```diff
@@ -6,40 +6,74 @@ import { Search } from './search/search';
 import { Resource } from './resource/resource';
 import { ResourceFilter } from './resource/resource_filter';
 
-const api = '/api/v1/resources/?format=json'
+let api = '/api/v1/resources/?format=json'
```

Comment:
Why did you change to `let`? From the docs I can see that:

`const` is a signal that the variable won’t be reassigned.

`let` is a signal that the variable may be reassigned

Reply:
Good catch, was gonna change this in a function but decided against it